sdk/cdn/Azure.ResourceManager.Cdn/src/autorest.md

```diff
@@ -6,16 +6,20 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 library-name: Cdn
 namespace: Azure.ResourceManager.Cdn
-require: 
-  - https://github.com/Azure/azure-rest-api-specs/blob/2cd7c6eacc5430d8956885e8d19b87ce3f3ebd6e/specification/cdn/resource-manager/readme.md
+title: CdnManagementClient
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/blob/2cd7c6eacc5430d8956885e8d19b87ce3f3ebd6e/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdn.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/2cd7c6eacc5430d8956885e8d19b87ce3f3ebd6e/specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdnwebapplicationfirewall.json
 clear-output-folder: true
 skip-csproj: true
 output-folder: Generated/
 operation-id-mappings:
   CdnEndpoint:
       profileName: Microsoft.Cdn/operationresults/profileresults
       endpointName: Microsoft.Cdn/operationresults/profileresults/endpointresults
-
+modelerfour:
+  naming:
+    preserve-uppercase-max-length: 2
```

Comment:
I remember some conversion about adding default configs for autorest.csharp here: https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/readme.md
We may consider add these as the default options as well.

Reply:
Good point, if we add this one to the default options, the `rename rule` configuration might be simpler.